pkg/app/ops/pipedstatsbuilder/builder.go

```diff
@@ -19,13 +19,18 @@ import (
 	"encoding/json"
 	"errors"
 	"io"
+	"time"
 
 	"go.uber.org/zap"
 
 	"github.com/pipe-cd/pipe/pkg/cache"
 	"github.com/pipe-cd/pipe/pkg/model"
 )
 
+const (
+	liveStateExceedDuration = 2 * time.Minute
```

Comment:
nit: How about something like:
```suggestion
	pipedStatsRetention = 2 * time.Minute
```

Reply:
Looks like It isn't only for liveState. And `retention` is often used for how long the data is stored in the monitoring context.

Reply:
Thanks, lets me apply this 🙏

Reply:
Addressed by https://github.com/pipe-cd/pipe/pull/2281/commits/bfccaa3ec393d757f1d0d5282c135469a1d0dcd9 🙏